wp-content/themes/csisnuclearnetwork/inc/template-functions.php

```diff
@@ -283,15 +283,22 @@ function nuclearnetwork_archive_titles( $title ) {
         $title = single_tag_title( '<span class="entry-header__title-label">Tag</span> ', false );
     } elseif ( is_tax() ) { //for custom post types
 			$title = sprintf( __( '%1$s' ), single_term_title( '', false ) );
-		} elseif (is_post_type_archive()) {
-				$title = post_type_archive_title( '', false );
-		} elseif ( is_author() ) {
-			$title = get_the_author();
-		}
+    } elseif (is_post_type_archive()) {
+            $title = post_type_archive_title( '', false );
+    } elseif ( is_author() ) {
+        $title = get_the_author();
+    }
     return $title;
 }
 add_filter( 'get_the_archive_title', 'nuclearnetwork_archive_titles' );
 
+function nuclearnetwork_taxonomy( $terms ) {
+    unset($terms[0]);
+  return $terms;
+}
+add_filter( 'term_links-filtered_content_types', 'nuclearnetwork_taxonomy' );
```

Comment:
This is close, but there's an issue when the parent term isn't selected.  For example, if the only content type selected is commentary and analysis isn't selected, the forward slash appears, but commentary doesn't. I think adding a check to see if `$terms[0]` matches the post type name should do the trick.